source/material_model/visco_plastic.cc

```diff
@@ -685,5 +685,12 @@ namespace aspect
                                    "The value for the components of this formula and additional "
                                    "parameters are read from the parameter file in subsection "
                                    " 'Material model/Visco Plastic'.")
+
+#define INSTANTIATE(dim) \
+  template class PlasticAdditionalOutputs<dim>;
+
+    ASPECT_INSTANTIATE(INSTANTIATE)
+
+#undef INSTANTIATE
```

Comment:
I think I know what happened. This code needs to go into the same source file that defines `PlasticAdditionalOutputs`. That is `source/material_model/rheology/visco_plastic.cc`, not this one. Please check that, and if true, move the code into the other file.

Reply:
It works!! Thank you!